iothub/device/src/Pipeline/RetryDelegatingHandler.cs

```diff
@@ -1231,6 +1238,25 @@ private void HandleConnectionStatusExceptions(Exception exception, bool retryAtt
                     nameof(HandleConnectionStatusExceptions));
         }
 
+        private bool IsClientTransportOpen()
+        {
+            return Interlocked.Read(ref _isOpen) == 1;
```

Comment:
Save 1 and 0 as class constants.

Reply:
How about I assign the values to the enum directly and then the helpers can be modified to:

```c#
        private bool IsClientTransportOpen()
        {
            return Interlocked.Read(ref _isOpen) == (int)ClientTransportStatus.Open;
        }

        private void SetClientTransportStatus(ClientTransportStatus transportStatus)
        {
            _ = Interlocked.Exchange(ref _isOpen, (int)transportStatus);
        }
```